server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -78,18 +78,23 @@ export default class ServiceProviderReferralsController {
 
   async showInterventionProgress(req: Request, res: Response): Promise<void> {
     const sentReferral = await this.interventionsService.getSentReferral(res.locals.user.token, req.params.id)
+    const serviceUserPromise = this.communityApiService.getServiceUserByCRN(sentReferral.referral.serviceUser.crn)
     const serviceCategoryPromise = this.interventionsService.getServiceCategory(
       res.locals.user.token,
       sentReferral.referral.serviceCategoryId
     )
     const actionPlanPromise =
       sentReferral.actionPlanId === null
         ? Promise.resolve(null)
-        : this.interventionsService.getActionPlan(res.locals.user.token, req.params.id)
+        : this.interventionsService.getActionPlan(res.locals.user.token, sentReferral.actionPlanId)
```

Comment:
this was a bug

Reply:
Yeah, I've got a [commit](https://github.com/ministryofjustice/hmpps-interventions-ui/pull/171/commits/e7acd120df655d7222df71442f5e6571bbbf7d3d) for this in my PR that explains it, perhaps we can merge that one first and then won't need this?

Reply:
sure